src/engine.py

```
@@ -871,6 +889,94 @@ def _parse_port_range(self, value: Optional[Any], fallback_port: int) -> Tuple[i
+            ]
+        if not allowed_remote_hosts:
+            allowed_remote_hosts = getattr(global_external, "allowed_remote_hosts", None)
+        if isinstance(allowed_remote_hosts, str):
+            allowed_remote_hosts = [
+                item.strip() for item in allowed_remote_hosts.split(",") if item.strip()
+            ]
+        if not allowed_remote_hosts:
+            try:
+                ipaddress.ip_address(str(self.config.asterisk.host))
+                allowed_remote_hosts = [str(self.config.asterisk.host)]
+            except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)